internal/scan-manager/scanners/common/pom.xml

```diff
@@ -40,60 +40,60 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
-            <version>1.5.6.RELEASE</version>
+            <version>${org.springframework.boot.starter.version}</version>
         </dependency>
         <dependency>
             <groupId>org.wso2.security.tools.scanmanager</groupId>
             <artifactId>scan-manager-common</artifactId>
-            <version>${org.wso2.security.tools.scanmanager.common.version}</version>
+            <version>${scanmanager.common.version}</version>
         </dependency>
         <dependency>
             <groupId>org.wso2.security.tools.scanmanager</groupId>
             <artifactId>scan-manager-common-internal</artifactId>
-            <version>${org.wso2.security.tools.scanmanager.common.internal.version}</version>
+            <version>${scanmanager.common.internal.version}</version>
         </dependency>
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.3</version>
+            <version>${org.apache.httpcomponents.httpclient.version}</version>
         </dependency>
         <dependency>
             <groupId>com.jcraft</groupId>
             <artifactId>jsch</artifactId>
-            <version>0.1.54</version>
+            <version>${com.jcraft.jsch.version}</version>
         </dependency>
         <dependency>
             <groupId>org.wso2.orbit.xerces</groupId>
             <artifactId>xercesImpl</artifactId>
-            <version>2.12.0.wso2v1</version>
+            <version>${org.wso2.orbit.xerces.version}</version>
         </dependency>
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-compress</artifactId>
-            <version>1.16</version>
+            <version>${org.apache.commons.version}</version>
         </dependency>
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
-            <version>1.24</version>
+            <version>${org.yaml.snakeyaml.version}</version>
         </dependency>
         <dependency>
             <groupId>log4j</groupId>
             <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
+            <version>${log4j.version}</version>
         </dependency>
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.8.2</version>
+            <version>${com.google.code.gson.version}</version>
         </dependency>
     </dependencies>
     <build>
         <plugins>
```

Comment:
Shall we please add `maven-java-doc` plugin to this project with this PR. Add the commented configuration at https://github.com/wso2/security-tools/blob/master/internal/pom.xml#L264

We can incrementally add it to other projects.